src/gmt_plot.c

```diff
@@ -2583,6 +2583,14 @@ GMT_LOCAL void gmtplot_consider_internal_annotations (struct GMT_CTRL *GMT, stru
 	PSL_settextmode (PSL, PSL_TXTMODE_HYPHEN);	/* Back to leave as is */
 }
 
+bool gmtplot_skip_pole_lat_annotation (struct GMT_CTRL *GMT, double lat) {
+	if (fabs (lat) < 90.0) return false;	/* Not at the poles so OK to annotate */
```

Comment:
```suggestion
	if (fabs (lat) < 86.0) return false;	/* Not at the poles so OK to annotate */
```
I was thinking of this type of situation which might motivate a cutoff < 90 (``gmt basemap -Rd -JKf12c -Ba11fg -png map``), but perhaps it should be allowed even if bad looking

Reply:
I agree that there is no simple way to place latitudes > 85 for -JKf so we should prevent it.  But only for this projection since the others that have the pole as a straight line have a much larger "take-off" angle at the end of that line so the algorithm I implemented works.  It is just -Jf that as such a tiny angle from 90 to 89 that we end up with small/tiny and it blows up.  So if you agree I will modify the check to do < 85 for Eckert 4 but not the others.

Reply:
Hm, actually, you may have a point.  85 does not look great for hammer either.

Reply:
Not out of woods: Here is what I see when testing with

`gmt basemap -Rd -JH100c -Ba2fg  -png map --MAP_ANNOT_MIN_SPACING=0 --FONT_ANNOT_PRIMARY=9p`

changing -J to the different misc. projections.

1. Mollweide looks passible up to 88 for me
2. Hammer, which is almost the same, does not.  It needs to stop at 85.
3. Eckert 6 is OK in the sense of the offset, but the y-change from 85 to 90 is so tiny it will overprint except for our new MIN_ANNOT_OFFSET setting.  Set that to 0 and it looks ugly.  I think that may be OK.
4. THe Robinson, WInkel, Sinusoidal, Van Grinten all look OK

There is of course some crowding with those I deem work OK, but that is handled by the min spacing part.  So I suggest adding these specific limits for Mollweide [88], Hammer [85], and Eckert 4 [85] with the others at 90.

Reply:
Yes, seems limited to those two projections.

Reply:
> Not out of woods: Here is what I see when testing with
> 
> `gmt basemap -Rd -JH100c -Ba2fg -png map --MAP_ANNOT_MIN_SPACING=0 --FONT_ANNOT_PRIMARY=9p`
> 
> changing -J to the different misc. projections.
> 
>     1. Mollweide looks passible up to 88 for me
> 
>     2. Hammer, which is almost the same, does not.  It needs to stop at 85.
> 
>     3. Eckert 6 is OK in the sense of the offset, but the y-change from 85 to 90 is so tiny it will overprint except for our new MIN_ANNOT_OFFSET setting.  Set that to 0 and it looks ugly.  I think that may be OK.
> 
>     4. THe Robinson, WInkel, Sinusoidal, Van Grinten all look OK
> 
> 
> There is of course some crowding with those I deem work OK, but that is handled by the min spacing part. So I suggest adding these specific limits for Mollweide [88], Hammer [85], and Eckert 4 [85] with the others at 90.

Seems fine.